### DISCUSSION TITLE
Incomplete sentence in nextjs docs

### DISCUSSION THREAD

**Discussion by omhoumz** [vercel/next.js#10938] [Help] [answered]
I think that this paragraph in the docs is incomplete: 

> [...] Time to first byte (TTFB) will be slower than getStaticProps because the server must compute the result on every request, and the result cannot be cached by a CDN without extra .

I wanted to open a PR but I don't know what the author wanted to say here.

Maybe @chibicode and @timneutkens, or anyone else, can answer this.

Here is the [link to the docs paragraph](https://nextjs.org/docs/basic-features/data-fetching#when-should-i-use-getserversideprops) and a screenshot:

<img width="560" alt="Nexjs docs" src="https://user-images.githubusercontent.com/7444066/76329695-32f66080-62ed-11ea-8ff1-2a0dc6be1073.png">

**Comment by Timer** [comment, marked as answer]
Fixed in https://github.com/zeit/next.js/pull/10941.

Thanks!
